### ISSUE TITLE
Location search

### ISSUE THREAD

**Issue by steinbro** [soundscape-community/soundscape-web-client#21]
Right now, users can only start beacons for locations that show up in the recent callout list. This is pretty limiting (it's the set of places that are already very close). At the very least, add the ability to craft links that encode a beacon, so that users can open/share a link with a pre-filled beacon they can start.

**Comment by sebastianovide** [comment]
Yes, this would be amazing. I had a chat with a blind association in Norway, and I concluded that the most valuable feature of SoundScape, which is not provided by other apps, is the spatial sound navigation (the sonar-like sound).

If users can click on an address so that the app can start navigation immediately, they could use it while using any other app at the same time.

I hope to have some time this weekend to investigate if this is possible using a PWA, or if it needs to be an actual app. In that case, we'll need to wrap it in Ionic or a similar framework to gain access to native features.
